package.json

```diff
@@ -59,15 +59,14 @@
   "packageManager": "pnpm@10.9.0",
   "pnpm": {
     "overrides": {
-      "@react-types/shared": "3.25.0",
```

Comment:
This was the main problem. There are types in a newer version that are being referenced in RAC packages that design-toolkit is utilizing